cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -74,13 +76,16 @@ def from_service_account_file(cls, filename, *args, **kwargs):
 
     from_service_account_json = from_service_account_file
 
-    def __init__(self,
-                 transport=None,
-                 channel=None,
-                 credentials=None,
-                 client_config=None,
-                 client_info=None,
-                 client_options=None):
+    def __init__(
+            self,
+            transport: Union[quantum_engine_service_grpc_transport.
```

Comment:
This union has only one element. I think you meant `Optional[...]`?

Reply:
From the function doc, I think I missed adding a `Callable` option to the union. Will fix.